src/commands/Minion/slayertask.ts

```diff
@@ -32,13 +34,42 @@ export default class extends BotCommand {
 		});
 	}
 
+	public getAlternateMonsterList(currentTask: SlayerTaskTable, assignedTask: AssignableSlayerTask | null) {
```

Comment:
Don't pass currentTask to this function. It adds an unneeded import dependency, and the purpose of the function can be entirely completed with just 'assignedTask.'

Reply:
Thanks, changed.